src/QuiqChatClient.js

```diff
@@ -153,6 +159,7 @@ class QuiqChatClient {
   stop = () => {
     disconnectSocket();
     this.initialized = false;
+    this.connected = false;
```

Comment:
@fredvollmer I couldn't remember if there was a reason we didn't do this before. I couldn't think of any, can you?